<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ParentWindow</class>
 <widget class="QMainWindow" name="ParentWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>885</width>
    <height>620</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>50</width>
    <height>50</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>500</width>
    <height>100</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>DataViz 0.0</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QTableWidget" name="tableWidget"/>
      </item>
      <item>
       <widget class="QMdiArea" name="WindowsManager">
        <property name="viewMode">
         <enum>QMdiArea::SubWindowView</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>885</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLoad_Dataset"/>
    <addaction name="actionDraw_Dataset"/>
   </widget>
   <widget class="QMenu" name="menuPlot">
    <property name="title">
     <string>Plot</string>
    </property>
    <addaction name="actionPlot_histogram"/>
   </widget>
   <widget class="QMenu" name="menuAnalysis">
    <property name="title">
     <string>Analysis</string>
    </property>
    <addaction name="actionFunction"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionHelp"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuPlot"/>
   <addaction name="menuAnalysis"/>
   <addaction name="menuHelp"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>true</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::AllToolBarAreas</set>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionLoad_Dataset"/>
   <addaction name="separator"/>
  </widget>
  <action name="actionLoad_Dataset">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/dataset.svg</normaloff>:/icons/dataset.svg</iconset>
   </property>
   <property name="text">
    <string>Load Dataset</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/aboutSymbol.svg</normaloff>:/icons/aboutSymbol.svg</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/helpSymbol.svg</normaloff>:/icons/helpSymbol.svg</iconset>
   </property>
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="action_Color">
   <property name="text">
    <string>Color</string>
   </property>
  </action>
  <action name="actionDraw_Dataset">
   <property name="icon">
    <iconset>
     <normaloff>C:/Users/Toavina/Downloads/296912_write_pen_drawing_writing_pencil_icon.svg</normaloff>C:/Users/Toavina/Downloads/296912_write_pen_drawing_writing_pencil_icon.svg</iconset>
   </property>
   <property name="text">
    <string>Draw Dataset</string>
   </property>
  </action>
  <action name="actionPlot_multiple_datasets">
   <property name="text">
    <string>Plot multiple datasets</string>
   </property>
  </action>
  <action name="actionFunction">
   <property name="icon">
    <iconset>
     <normaloff>C:/Users/Toavina/Downloads/function-mathematical-symbol-svgrepo-com.svg</normaloff>C:/Users/Toavina/Downloads/function-mathematical-symbol-svgrepo-com.svg</iconset>
   </property>
   <property name="text">
    <string>Function</string>
   </property>
  </action>
  <action name="actionPlot_histogram">
   <property name="icon">
    <iconset>
     <normaloff>C:/Users/Toavina/Downloads/histogram-svgrepo-com.svg</normaloff>C:/Users/Toavina/Downloads/histogram-svgrepo-com.svg</iconset>
   </property>
   <property name="text">
    <string>Plot histogram</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
